src/interfaces/userInterface.ts

```diff
@@ -0,0 +1,13 @@
+export interface userProps {
+  id: number;
+  name: string;
+  birthDate: string;
+  position: string;
+  salary: number;
+  projects: string[];
+  imageUrl: string;
+}
```

Comment:
Are all of these properties always returning values, or are some that are optional? Please, if there are optional values, type them accordingly - example -> salary?: number